argoverse/utils/make_att_files.py

```diff
@@ -304,27 +306,22 @@ def compute_v_a(traj: np.array) -> Tuple[np.array, np.array]:  # traj:Nx3
                     ind_close_max = ind_close.max() + 1
                     ind_close_min = ind_close.min()
 
-                if dict_tracks[id_track]["length_track"] == 0:
-                    # this shouldn't happen
-                    dict_tracks[id_track]["difficult_att"].append("zero_length")
-
+                if dict_tracks[id_track]["list_dist"][ind_valid].min() > dist_close:
+                    dict_tracks[id_track]["difficult_att"].append("far")
                 else:
-                    if dict_tracks[id_track]["list_dist"][ind_valid].min() > dist_close:
-                        dict_tracks[id_track]["difficult_att"].append("far")
+                    if dict_tracks[id_track]["length_track"] < 10 or dict_tracks[id_track]["exists"].sum() < 10:
```

Comment:
can we make 10 a module-level private variable? e.g. `NEAR_DISTANCE_THRESH_ = 10`

Reply:
It's not a distance threshold but a time threshold. I added a variable for it.